odoo/tests/common.py

```diff
@@ -1637,6 +1651,8 @@ def authenticate(self, user, password):
             # than this transaction.
             self.cr.flush()
             self.cr.clear()
+            if not password:
```

Comment:
Nitpicking but I don't want an empty string to behave weird
```suggestion
            if password is None:
```